apprating/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
```

Comment:
we should be adding this line to all new modules: 
apply from: "../jacoco.gradle"
to be able to run jacoco coverage task for this module